### PR TITLE
Add options to limit libass glyph and bitmap caches, and to control target rendering FPS

### PR DESCRIPTION
**Pull request by JustAMan**

### CODE REVIEW

**Comment by JustAMan** [comment]
Note: this fixes https://github.com/Dador/JavascriptSubtitlesOctopus/issues/12

File: src/subtitles-octopus.js [hunk 1]
```diff
@@ -28,7 +31,7 @@ var SubtitlesOctopus = function (options) {
     self.subUrl = options.subUrl; // Link to sub file (optional if subContent specified)
     self.subContent = options.subContent || null; // Sub content (optional if subUrl specified)
     self.onErrorEvent = options.onError; // Function called in case of critical error meaning sub wouldn't be shown and you should use alternative method (for instance it occurs if browser doesn't support web workers).
-    self.debug = options.debug || false; // When debug enabled, some performance info printed in console.
+    self.debug = options.debug || false;  // When debug enabled, some performance info printed in console.
```
Comment (dkanada) [inline src/subtitles-octopus.js]:
```suggestion
    self.debug = options.debug || false; // When debug enabled, some performance info printed in console.
```

File: src/subtitles-octopus.js [hunk 2]
```diff
@@ -253,9 +259,9 @@ var SubtitlesOctopus = function (options) {
         }
         if (self.debug) {
             var drawTime = Math.round(performance.now() - beforeDrawTime);
-            var blendTime = data.blendTime || 0;
-            if (blendTime > 0) {
-                console.log('render: ' + Math.round(data.spentTime - blendTime) + ' ms, blend: ' + Math.round(blendTime) + ' ms, draw: ' + drawTime + ' ms');
+            var blendTime = data.blendTime;
```
Comment (EraYaN) [inline src/subtitles-octopus.js:262]: Is this an unrelated change?

**Review by EraYaN — APPROVED** [review]

File: src/subtitles-octopus.js (hunk 2 above)
Comment (JustAMan) [reply src/subtitles-octopus.js:262]: Yes, a small one to slightly adjust the debug output :)

**Comment by Artiume** [comment]
Should we backport?

**Comment by JustAMan** [comment]
Backport where? If you mean back to upstream I'm working on it.
I think this particular PR has already made its way in as part of https://github.com/Dador/JavascriptSubtitlesOctopus/pull/66
